modules/ocf/manifests/apt.pp

```diff
@@ -2,7 +2,13 @@
   package { ['aptitude', 'imvirt', 'apt-transport-https', 'lsb-release']:; }
 
   class { '::apt':
-    purge => {
+    # Run apt update at every Puppet run.
+    update => {
+      frequency => 'always',
+      loglevel  => 'debug',
```

Comment:
we want debug logs?

Reply:
By default, the log level would be notify, which according to the docs causes puppet to report the update as a significant change.